### PR TITLE
[9.0][MIG] Migration of module stock_auto_move to v9.0

### PR DESCRIPTION
**Pull request by zakiuu**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
Thank you @zakiuu. The code LGTM. 
I do not feel comfortable with the change to a fixed procurement group for automatic moves and I don't understand the rational behind this change.  Maybe @npiganeau could enlighten me on the motivations behind this change of procurement group. IMO automatic moves should stay into their original procurement group since the original group can have a strong functional semantic. If you change the group, we lost this semantic. This change is maybe useful in some cases but I would prefer it to be optional. 
Nevertheless since the scope of this PR is the migration of this module to 9.0 :+1:

**Comment by npiganeau** [comment]
@lmignon Original reason is explained here #180, but since then even in this case we don't need this "feature" anymore. So I now agree with you, procurement group should not be modified.

**Comment by lmignon** [comment]
@npiganeau Thank you for explanation! @zakiuu Can you propose a PR to remove this feature or at least make it optional.

**Comment by zakiuu** [comment]
@lmignon yeah sure

**Review by rousseldenis — APPROVED** [review]
Code review. Agree with comments about procurement.group

**Comment by zakiuu** [comment]
@lmignon @rousseldenis @npiganeau The PR that fixes procurement group has been created #364

**Review by lmignon — APPROVED** [review]

**Comment by lmignon** [comment]
ping @Cedric-Pigeon

**Comment by zakiuu** [comment]
ping @Cedric-Pigeon

File: stock_auto_move/README.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Automatic Move Processing
+=========================
+
+This modules adds the possibility to have move automatically processed as soon
```
Comment (ThomasBinsfeld) [inline stock_auto_move/README.rst]: @zakiuu s/modules/module

File: stock_auto_move/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+To configure this module, you need to:
+
+* Set the flag "Automatic" on procurement rule
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/8.0
```
Comment (ThomasBinsfeld) [inline stock_auto_move/README.rst]: @zakiuu s/8.0/9.0

File: stock_auto_move/README.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Nicolas Piganeau <nicolas.piganeau@ndp-systemes.fr>
```
Comment (ThomasBinsfeld) [inline stock_auto_move/README.rst]: @zakiuu Add yourself in contributors

File: stock_auto_move/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (ThomasBinsfeld) [inline stock_auto_move/__openerp__.py]: @zakiuu Replace the headers by the [short ones](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py)

File: stock_auto_move/tests/test_stock_auto_move.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (ThomasBinsfeld) [inline stock_auto_move/tests/test_stock_auto_move.py]: @zakiuu Header

**Review by ThomasBinsfeld — APPROVED** [review]
Nitpicking :+1:

File: stock_auto_move/tests/test_stock_auto_move.py (hunk 5 above)
Comment (lmignon) [inline stock_auto_move/tests/test_stock_auto_move.py:17]: @zakiuu Can you check the last comments?

**Comment by zakiuu** [comment]
@lmignon done!

**Comment by zakiuu** [comment]
ping @Cedric-Pigeon

**Review by ThomasBinsfeld — APPROVED** [review]
:+1:

**Comment by zakiuu** [comment]
@rousseldenis @npiganeau  @lmignon  @sbidoul I guess this one is ready to be merged

**Comment by pedrobaeza** [comment]
Please squash together all the OCA Transbot commits

**Comment by zakiuu** [comment]
@pedrobaeza done!

**Pull request merged**